src/interfaces/coral_web/src/components/ReadOnlyConversation.tsx

```diff
@@ -0,0 +1,203 @@
+import { ChatMessage, isFulfilledMessage } from '@/types/message';
+import { cn, pluralize } from '@/utils';
+
+type Props = {
+  title: string;
+  messages: ChatMessage[];
+};
+
+/**
+ * @description Read only view of a shared conversation
+ */
+export const ReadOnlyConversation: React.FC<Props> = ({ title, messages }) => {
+  const {
+    citations: { citationReferences, selectedCitation },
+    selectCitation,
+  } = useCitationsStore();
+  const hasCitations = Object.keys(citationReferences).length > 0;
+
+  const handleClickOutside = useCallback(
+    (event: MouseEvent) => {
+      if (!selectedCitation) return;
+
+      const target = event.target as Node;
+      const invalidElements = Array.from(
+        document.querySelectorAll(`.${ReservedClasses.MESSAGE}, .${ReservedClasses.CITATION}`)
+      );
+
+      const isClickInsideInvalidElements = invalidElements.some((node) => node.contains(target));
+      if (!isClickInsideInvalidElements) {
+        selectCitation(null);
+      }
+    },
+    [selectedCitation, selectCitation]
+  );
+
+  useEffect(() => {
+    window?.addEventListener('click', handleClickOutside);
+    return () => {
+      window?.removeEventListener('click', handleClickOutside);
+    };
+  }, [handleClickOutside]);
+
+  return (
+    <>
+      <PageHead title={title} />
+      <div
+        className={cn('flex w-full flex-col gap-2 pb-28 pt-12 md:px-5', {
+          'max-w-share-content': !hasCitations,
+          'max-w-share-content-with-citations': hasCitations,
+        })}
+      >
+        <Text styleAs="h3" className="text-center text-volcanic-800">
+          {title}
+        </Text>
+
+        <div className={cn('my-6 w-full border-b border-marble-500')} />
+
+        <div className="flex flex-col gap-y-4 py-6 md:gap-y-6">
+          {messages.map((m, i) => (
+            <div key={i} className="flex items-start gap-x-3">
+              <MessageRow
+                message={m}
+                isLast={messages.length - 1 === i}
+                isStreamingToolEvents={false}
+                className={cn('max-w-full md:max-w-[80%]', { 'md:max-w-full': !hasCitations })}
+              />
+              {isFulfilledMessage(m) && m.citations && m.citations.length > 0 && (
+                <ReadOnlyConversationCitation message={m.text} generationId={m.generationId} />
+              )}
+            </div>
+          ))}
+        </div>
+      </div>
+      <div className="fixed bottom-0 left-0 z-read-only-conversation-footer flex w-full items-center justify-center bg-white py-4 shadow-top">
+        <Button label="Start a new conversation" href="/" splitIcon="arrow-right" kind="primary" />
+      </div>
+    </>
+  );
+};
+
+type ReadOnlyConversationCitationProps = {
+  generationId: string;
+  message: string;
+};
+
+/**
+ * @description Citation for read only view of a shared conversation with simplified positioning
+ */
+const ReadOnlyConversationCitation: React.FC<ReadOnlyConversationCitationProps> = ({
```

Comment:
Simplified version of `Citation.tsx`
- Aligns with the top of the message row
- Does not jump next to the selected highlighted text in the message

Reply:
feel free to put this in the description